src/OpenDebugAD7/VariableManager.cs

```diff
@@ -56,6 +77,24 @@ internal int Create(VariableScope scope)
             return m_variableHandles.Create(scope);
         }
 
+        public void AddChildVariable(int parentHandle, IDebugProperty2 prop)
```

Comment:
>IDebugProperty2 prop [](http://example.com/codeflow?start=55&length=20)

You might want to pass in a property info instead of a property, as the caller would already have the name